src/java/org/apache/cassandra/metrics/TableMetrics.java

```diff
@@ -204,6 +204,8 @@
     public final Counter speculativeInsufficientReplicas;
     public final Gauge<Long> speculativeSampleLatencyNanos;
 
+    public final Counter transientWrites;
```

Comment:
This doesn't seem so well named? I mean it's really not just writes to transients it's rapid write protection. We call them speculative retries on the read path.